src/workflow/trinary_input.c

```diff
@@ -35,12 +38,28 @@ static void _confirm(const char* word, void* param)
     workflow_blocking_unblock();
 }
 
+static void _select(uint8_t idx)
+{
+    ui_screen_stack_pop();
+    if (idx == 0) {
+        _cancel_reason = WORKFLOW_TRINARY_INPUT_RESULT_DELETE;
+        workflow_cancel_force();
+    } else if (idx == 1) {
+        _cancel_reason = WORKFLOW_TRINARY_INPUT_RESULT_CANCEL;
+        workflow_cancel();
+    }
+}
```

Comment:
Blank line to separate two functions? I wonder why clang lint didn't catch it.

Reply:
done